build/build-treemap.js

```diff
@@ -28,6 +28,8 @@ async function run() {
       fs.readFileSync(require.resolve('tabulator-tables/dist/js/modules/format.js'), 'utf8'),
       fs.readFileSync(require.resolve('tabulator-tables/dist/js/modules/resize_columns.js'), 'utf8'),
       /* eslint-enable max-len */
+      {path: '../../lighthouse-core/report/html/renderer/i18n.js'},
+      {path: '../../lighthouse-core/report/html/renderer/util.js'},
```

Comment:
I only added util.js to get at some of the formatting methods. Maybe those should be moved to i18n.js? well, we also keep all the i18n string data in Util...so a lot would have to move to i18n.js . worth?